### PR TITLE
#1009 add counter documentation

### PR DESCRIPTION
**Pull request by saadmhybris**
Closes sap/fundamental#1009

added counter documentation

#### Test

* http://localhost:4000/components/badge-counter-label-status.html#default-counter

#### Changelog

**New**

* counter documentation

**Changed**

nothing 

**Removed**

nothing

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit 81905504cb063b4ccf306b6326520091d8f81a41

https://deploy-preview-1063--fundamental.netlify.com

**Comment by xak** [comment]
A few things ...

- Add subheads to break these apart, in this order ... Badge, Label, Status, Counter. They all run together.
- "Semantic" not "symentic" — http://localhost:4000/components/badge-counter-label-status.html#status-indicator-label-with-symentic-colors
- Sidebar link is broken — http://localhost:4000/components/badge-label-status.html
- Components page is broken — 
http://localhost:4000/components/badge-label-status.html

File: docs/_data/components.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
       meta: alert
 
     - title: Badge, Status & Label
```
Comment (xak) [inline docs/_data/components.yml]: How about just call this "Status Indicators"?

**Comment by saadmhybris** [comment]
@xak ready for review again 

http://localhost:4000/components/status-indicators.html

File: docs/pages/components/status-indicators.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
+---
+title: Status Indicators
+keywords: status indicators, status, indicators, badge, label, counter
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/status-indicators.html
+folder: components
+---
+
+Status Indicators are used to easily highlight the state of an object. `badge`, `label`, `status-label`
```
Comment (xak) [inline docs/pages/components/status-indicators.md]: Add `counter` here.

File: docs/pages/components/status-indicators.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+<hr />
+
+# Badge
+
+## Badge Style Options
+`badge` Status Indicator has three options: `defautl`, `--pill` and `--filled`
```
Comment (xak) [inline docs/pages/components/status-indicators.md]: `default` not `defautl`

**Pull request merged**